packages/nimble-components/src/rich-text-editor/tests/rich-text-editor-matrix.stories.ts

```diff
@@ -64,6 +86,77 @@ export const richTextEditorSizing: StoryFn = createStory(html`
     ])}
 `);
 
+const styledComponent = (): ViewTemplate => html`
+<div style="padding: 20px">
+    <${richTextEditorTag}>
+        <${buttonTag} slot="footer-actions" appearance="ghost">Cancel</${buttonTag}>
+        <${buttonTag} slot="footer-actions" appearance="outline">Ok</${buttonTag}>
+    </${richTextEditorTag}>
+</div>
+`;
+
+const mobileWidthComponent = (): ViewTemplate => html`
+<div style="padding: 20px; width: 300px;">
+    <${richTextEditorTag}>
+        <${buttonTag} slot="footer-actions" appearance="ghost">Cancel</${buttonTag}>
+        <${buttonTag} slot="footer-actions" appearance="outline">Ok</${buttonTag}>
+    </${richTextEditorTag}>
+</div>
+`;
+
+export const editorWhenMarkdownValueSetInLightThemeWhiteBackground: StoryFn = createFixedThemeStory(
+    createMatrix(styledComponent),
+    lightThemeWhiteBackground
+);
+
+editorWhenMarkdownValueSetInLightThemeWhiteBackground.play = playFunction;
+
+export const editorWhenMarkdownValueSetInColorThemeDarkGreenBackground: StoryFn = createFixedThemeStory(
+    createMatrix(styledComponent),
+    colorThemeDarkGreenBackground
+);
+
+editorWhenMarkdownValueSetInColorThemeDarkGreenBackground.play = playFunction;
+
+export const editorWhenMarkdownValueSetInDarkThemeBlackBackground: StoryFn = createFixedThemeStory(
+    createMatrix(styledComponent),
+    darkThemeBlackBackground
+);
+
+editorWhenMarkdownValueSetInDarkThemeBlackBackground.play = playFunction;
```

Comment:
You should be able to use `createMatrixThemeStory` to render all three of these themes in the same story. You'll have to modify your `playFunction` to get all editors instead of just one.

Reply:
Updated as per your suggestion.